Formula/llvm.rb

```diff
@@ -133,7 +160,7 @@ def install
       # Workaround for CMake Error: failed to create symbolic link
       ENV.deparallelize if Hardware::CPU.arm?
       system "cmake", "--build", "."
-      system "cmake", "--build", ".", "--target", "install"
+      system "cmake", "--build", ".", "--target", "install/strip"
       system "cmake", "--build", ".", "--target", "install-xcode-toolchain" if MacOS::Xcode.installed?
```

Comment:
Might be good to use `install-xcode-toolchain/strip` here too. If there is such a target, that is.